PocketKit/Sources/PocketKit/Home/HomeViewModel.swift

```diff
@@ -126,15 +135,21 @@ extension HomeViewModel {
         }
 
         slates.forEach { slate in
+            // Create a topic and slate section _only_ if any
+            // recommendations exist for the given slate
+            let recs = slate.recommendations?
```

Comment:
It's unfortunate that these core data relationships aren't sets with proper generic types, forcing us to iterate over each item and downcast to the related item type :/

What do you think about adding an extension to `Slate` that encapsulates the downcasting?

Something like
```swift
extension Slate {
  // open to different names here
  // but need to avoid collision with the existing `recommendations` property
  var recommendationsArray: [Recommendation] {
    recommendations.compactMap { $0 as? Recommendation }
  }
}
```